Odata-docs/changelog/odataconnectedservice.md

```diff
@@ -9,7 +9,62 @@ ms.topic: article
 
 # OData Connected Service changelog
 
-OData Connected Service extension is available on [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=laylaliu.ODataConnectedService).
+OData Connected Service extension is available on [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=marketplace.ODataConnectedService).
+
+## OData Connected Service 0.12.1
+
+### Improvements and Bug Fixes:
+* [ [#202](https://github.com/OData/ODataConnectedService/pull/202) ] Fix invalid escaping of double quotes when generating VB.NET client proxy
+* [ [#204](https://github.com/OData/ODataConnectedService/pull/204) ] Fix invalid namespace emitted when generating client proxy with custom namespace supplied
+* [ [#209](https://github.com/OData/ODataConnectedService/pull/209) ] Restore namespace in the metadata document as the default namespace applied in the client proxy
+* [ [#203](https://github.com/OData/ODataConnectedService/pull/203) ] Fix non-compilable code emitted when the option to generate multiple files is checked and the model contains 2 types with the same name but in different namespace
```

Comment:
Should these be in any particular order? (alphabetical, pullId, date of merge, etc?)
If not lgtm, if yes, probably applies for all the other entries...

Reply:
@g2mula Traditionally we have never bothered about any order. Alphabetic would be a bit hard to manage since descriptions tend to mutate often. Date of merge would be tedious to keep track of. We could maybe adopt pull request ID to create some order, and since it doesn't hurt I'll make that change